tests/e2e/env/setup.sh

```diff
@@ -324,12 +324,6 @@ else
 	echo "Skipping install of Action Scheduler"
 fi
 
-if [[ ! ${SKIP_WC_BLOCKS_TESTS} ]]; then
```

Comment:
If we're removing this, should we remove references to it as it no longer does anything.

Reply:
For this PR, I think it is best to leave in this env var because it is used to filter out some test suites, e.g:

https://github.com/Automattic/woocommerce-payments/blob/adae84b8d870865befd409f1466edbeef7993740/tests/e2e/utils/flows.js#L60

https://github.com/Automattic/woocommerce-payments/blob/adae84b8d870865befd409f1466edbeef7993740/tests/e2e/specs/blocks/shopper/shopper-wc-blocks-saved-card-checkout-and-usage.spec.js#L27